inspector/scripts/inspect-beacon-group.js

```diff
@@ -0,0 +1,87 @@
+const truffleContract = require("@truffle/contract")
+const KeepRandomBeaconOperatorJson = require("@keep-network/keep-core/artifacts/KeepRandomBeaconOperator.json")
+
+module.exports = async function() {
+    const deploymentBlock = 10834116
+
+    try {
+        const KeepRandomBeaconOperator = truffleContract(KeepRandomBeaconOperatorJson)
+        KeepRandomBeaconOperator.setProvider(web3.currentProvider)
+
+        const keepRandomBeaconOperator = await KeepRandomBeaconOperator.deployed()
+
+        const numberOfGroups = await keepRandomBeaconOperator.numberOfGroups()
+        const entryRequestedEvents = await keepRandomBeaconOperator.getPastEvents(
+            "RelayEntryRequested",
+            {
+                fromBlock: deploymentBlock,
+                toBlock: "latest",
+            }
+        )
+        const entrySubmittedEvents = await keepRandomBeaconOperator.getPastEvents(
+            "RelayEntrySubmitted",
+            {
+                fromBlock: deploymentBlock,
+                toBlock: "latest",
+            }
+        )
+        const timeoutEvents = await keepRandomBeaconOperator.getPastEvents(
+            "RelayEntryTimeoutReported",
+            {
+                fromBlock: deploymentBlock,
+                toBlock: "latest",
+            }
+        )
+
+        console.log(`Number of groups:            ${numberOfGroups}`)
+        console.log(`Relay entries requested:     ${entryRequestedEvents.length}`)
+        console.log(`Relay entries submitted:     ${entrySubmittedEvents.length}`)
+        console.log(`Number of timed-out entries: ${timeoutEvents.length}`)
+        if (timeoutEvents.length > 0) {
+            console.log(`timed out entry group indices: ${timeoutEvents.map(event => event.returnValues.groupIndex)}`)
+        }
```

Comment:
Right, this [event](https://github.com/keep-network/keep-core/blob/master/solidity/contracts/KeepRandomBeaconOperator.sol#L596) is emitted right after the timeout happened, but it is still the same index. You sure we don't need these lines?

Reply:
It is not the same index, unfortunately. The index is changed in `signRelayEntry` a few lines earlier, see [here](https://github.com/keep-network/keep-core/blob/master/solidity/contracts/KeepRandomBeaconOperator.sol#L427). Because of that, the information "timed out entry group indices" is not correct.

Reply:
Okay, I missed `signRelayEntry` part. console.log is removed